internal/provider/iam/data_user_principal_test.go

```diff
@@ -19,23 +19,23 @@ func TestAccUserPrincipalDataSource(t *testing.T) {
 		PreCheck:                 func() { acctest.PreCheck(t) },
 		Steps: []resource.TestStep{
 			{
-				Config: testAccUserPrincipalConfigEmailInput("cloud-experiences-tooling@hashicorp.com"),
+				Config: testAccUserPrincipalConfigEmailInput("hcp-terraform-provider-iam@hashicorp.com"),
```

Comment:
Thanks for changing this email! Will it be available in 1Password as well in case things need to be debugged?

Reply:
Yep! The new login creds for this org are in 1Pass. I can also invite you to manage the org if you'd like